cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/handlers/AppLifecycleHttpHandlerTest.java

```diff
@@ -142,12 +143,12 @@ public ApplicationLifecycleService createLifeCycleService(CConfiguration cConf,
           MetadataServiceClient metadataServiceClient,
           AccessEnforcer accessEnforcer, AuthenticationContext authenticationContext,
           MessagingService messagingService, Impersonator impersonator,
-          CapabilityReader capabilityReader) {
```

Comment:
what is this change used for ?

Reply:
We are using transaction runner in ApplicationLifecycleService for scan method. Here we are creating an instance of ApplicationLifecycleservice for injection and it needs transaction runner as one of the args